### PR TITLE
Go version upgrade.

### PR DESCRIPTION
**Pull request by bravebeaver**
- upgrade go to 1.19
- bump helm version

### CODE REVIEW

**Review by Tatsinnit — COMMENTED** [review]
☕️🙏💡 suggestion / idea share: since we are updating the go-version let’s please consider updating to latest go version https://go.dev/doc/go1.19

thanks 🙏

File: .github/workflows/ci-pipeline.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ on:
     branches: [ master ]
 
 env:
-  GO_VERSION: '1.16.1'
+  GO_VERSION: '1.18'
```
Comment (peterbom) [inline .github/workflows/ci-pipeline.yaml:11]: I see that this previously included the minor version. Any thoughts on whether we should be continuing to do that (e.g. 1.18.9, or 1.19.4 if we're going for the latest stable one)? I'd guess it'd make the CI builds more consistent.

Comment (bravebeaver) [reply .github/workflows/ci-pipeline.yaml:11]: convinced. stability over "main version tag". will update

**Comment by Tatsinnit** [comment]
💡 Thanks for this, I noticed something so mentioning : we can open PR from your fork - feature branch, why have we this `go1.18` from azure org level branch PR?

https://github.com/Azure/aks-periscope/tree/go1.18

Screenshot 2023-01-11 at 12 31 28 PM

**Comment by bravebeaver** [comment]
close this one as a new PR is created following the correct fork model. thanks @Tatsinnit for pointing it out.

**Pull request closed without merging**